internal/action/edit.go

```diff
@@ -61,7 +63,10 @@ func (s *Action) editUpdate(ctx context.Context, name string, content, nContent
 
 	// write result (back) to store.
 	if err := s.Store.Set(ctxutil.WithCommitMessage(ctx, fmt.Sprintf("Edited with %s", ed)), name, nSec); err != nil {
-		return exit.Error(exit.Encrypt, err, "failed to encrypt secret %s: %s", name, err)
+		if !errors.Is(err, store.ErrMeaninglessWrite) {
+			return exit.Error(exit.Encrypt, err, "failed to encrypt secret %s: %s", name, err)
+		}
+		out.Warningf(ctx, "The new value of the password is equal to its current value. Are you sure you want this?")
```

Comment:
This is a question without a choice. If we do not want to give the user a choice we shouldn't include a question in the first place.